junit-vintage-engine/src/testFixtures/java/org/junit/vintage/engine/samples/junit4/package-info.java

```diff
@@ -1,4 +1,13 @@
 
+/*
+ * Copyright 2015-2025 the original author or authors.
```

Comment:
is junit-vintage-engine something that needs licence as well?

@marcphilipp

just asking you from karlsruhe to karlsruhe, not to bother with big discussion topic. 

If not its all good. Thanks for correction/guidance.